Solutions/exercicio01_marzani/app/utils/s3_instance.py

```diff
@@ -0,0 +1,20 @@
+import boto3
+import logging
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+# Instanciando um cliente do S3.
+def instanciar_s3():
```

Comment:
As tuas funções de instanciação do client estão duplicadas, poderias ter apenas uma implementação e passar como parâmetro o recurso desejado.
```suggestion
def instantiate_client(resource: str):
```